src/components/PollViewer/PollDraftHandler.vue

```diff
@@ -0,0 +1,96 @@
+<!--
+  - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<NcDialog class="drafts"
+		:name="t('spreed', 'Poll drafts')"
+		size="large"
+		close-on-click-outside
+		v-on="$listeners"
+		@update:open="emit('close')">
+		<EmptyView v-if="!pollDrafts.length"
+			class="drafts__empty"
+			:name="t('spreed', 'No poll drafts')"
+			:description="t('spreed', 'There is no poll drafts yet saved for this conversation')">
+			<template #icon>
+				<IconPoll />
+			</template>
+		</EmptyView>
+		<div v-else class="drafts__wrapper">
+			<Poll v-for="item in pollDrafts"
+				:id="item.id.toString()"
+				:key="item.id"
+				:token="token"
+				:name="item.question"
+				draft
+				@click="openPollEditor" />
+		</div>
+		<template v-if="props.showCreateButton" #actions>
+			<NcButton @click="openPollEditor(null)">
+				{{ t('spreed', 'Create new poll') }}
+			</NcButton>
```

Comment:
So, this is basically shown when opening from shared items tab, it will also need  a back button so the flow could be completed and revertible. 

I am not sure if it makes sense to keep the browse draft button too because you were just in the drafts handler before clicking "create new poll", it's intuitive to go back and select from there.